datadog_cluster_agent/assets/dashboards/datadog_cluster_agent_overview.json

```
@@ -0,0 +1,418 @@
+{
+    "author_info": {
+        "author_name": "Datadog"
+    },
+    "board_title": "Datadog Cluster Agent - Overview",
```

```suggestion
    "board_title": "Datadog Cluster Agent Overview",
```

nit: on syncing, the board title is rendered in the url like `/datadog-cluster-agent--overview`